polynote-spark/src/test/scala/polynote/server/NotebookRunningSpec.scala

```diff
@@ -50,14 +50,14 @@ class NotebookRunningSpec extends FreeSpec with Matchers  {
       1 -> List(ResultValue("Out","Int",List(StringRepr("2"), DataRepr(IntType,null)),1.toShort,2,null,Some((0,0)))),
       2 -> List(Output("text/plain; rel=stdout","three\n")),
       3 -> List(Output("text/html","<strong>HTML!</strong>")),
-      4 -> List(ResultValue("x","Int",List(StringRepr("2"), DataRepr(IntType,null)),4.toShort,2,null,Some((4,4)))),
+      4 -> List(ResultValue("x","Int",List(StringRepr("2"), DataRepr(IntType,null)),4.toShort,2,null,Some((0,0)))),
```

Comment:
Assuming these positions are wrong now? Trying to think of what the best way is to remember to fix this later... should we leave the positions correct in the test and just let it fail?

Reply:
good point... I'd rather not have a failing test - we could have a TODO (assuming that we'll get to it sooner rather than later because the positioning being messed up will be obvious?). Or, we could fix the positioning :P